code/modules/vore/resizing/resize_vr.dm

```diff
@@ -272,15 +272,17 @@ var/const/RESIZE_A_SMALLTINY = (RESIZE_SMALL + RESIZE_TINY) / 2
 				// User is a human (capable of scooping) and not wearing shoes! Scoop into foot slot!
 				equip_to_slot_if_possible(tmob.get_scooped(H), slot_shoes, 0, 1)
 				if(istype(H.tail_style, /datum/sprite_accessory/tail/taur/naga))
-					src << "You wrap up [tmob] with your powerful tail!"
-					tmob << "[src] binds you with their powerful tail!"
+					src << "You slither over [tmob] with your large, thick tail, smushing them against the ground before coiling up around them, trapping them within the tight coils of your tail!"
```

Comment:
Repetitive use of the word "coil" annoys me.

Reply:
Changed the second coil to confines.